_config.yml

```diff
@@ -21,7 +21,7 @@ plugins:             [jekyll-paginate, jekyll-sitemap, jekyll-feed, jekyll-seo-t
 
 # Customise atom feed settings (this is where Jekyll-Feed gets configuration information)
 title:		           'LFTCM 2020'
-description:	       'Lean for the curious mathematicians 2020'
+description:	       'Lean for the Curious Mathematician 2020'
```

Comment:
I think this is what shows up at the top of the page but I'm not 100% sure, didn't build locally.